src/go/rpk/pkg/cli/redpanda/admin/config/config.go

```diff
@@ -70,10 +73,71 @@ func newLogLevelCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 	}
 	cmd.AddCommand(
 		newLogLevelSetCommand(fs, p),
+		newLogLevelGetCommand(fs, p),
 	)
 	return cmd
 }
 
+func newLogLevelGetCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var host string
+
+	cmd := &cobra.Command{
+		Use:   "get [LOGGERS...]",
+		Short: "Get broker logger's log level",
+		Long: `Get broker logger's log level.
+
+This command obtains the broker logger's log level. Each Redpanda
+broker has many loggers, and each can be individually returned.
+
+You can specify many loggers at once by separating each logger name with a space.
+To return all loggers, you can leave the logger argument empty or pass "all". 
+To see all possible loggers, run 'redpanda --help-loggers' in your node.
+`,
+
+		Example: `  # List log levels for raft and kafka
+  rpk redpanda admin config log-level get kafka raft --host localhost:9644
+`,
+		Args: cobra.MinimumNArgs(0), // <-- Corrected line
```

Comment:
I think the comment is a leftover 🤔 

Also, a nit, not worth changing, but there is [cobra.NoArgs](https://pkg.go.dev/github.com/spf13/cobra#NoArgs) to make it 'clearer' but effectively does the same 👍